Part3-Softwarequality_without_testing.md

```diff
@@ -119,7 +120,7 @@ Or do you have variables like
 
 or methods with 400 lines of code where you have absolutely no clue what happens with an input and even less idea what the output of the method is?
 
-To write code is your craftmanship, don't treat this lightly. A chef will keep his knives always sharp and clean. A developer must do the same. The book Clean Code by Robert C. Martin must read, but more importantly has to understand and then use everybody who works with code. Get involved in meetups and discussion about clean code practises, your code should be reviewed, and you must take the feedback and learn from it. Clean Code and your mindset around it, has to evolve. As a rule of thumb, if you look at code you wrote 6 months ago and you wouldn't change a thing, then you are stuck in your personal development, which keeps you from being an awesome developer.
+To write code is your craftmanship, don't treat this lightly. A chef will keep his knives always sharp and clean. A developer must do the same. <!-- The book Clean Code by Robert C. Martin must read, but more importantly has to understand and then use everybody who works with code. -->Get involved in meetups and discussion about clean code practices, your code should be reviewed, and you must take the feedback and learn from it. Clean Code and your mindset around it, has to evolve. As a rule of thumb, if you look at code you wrote 6 months ago and you wouldn't change a thing, then you are stuck in your personal development, which keeps you from being an awesome developer.
 Plan your implementation with the SOLID principles in mind:
 
 * **S**ingle Responsibility Principle
```

Comment:
Just listing the principles without any explaination is not really providing additional insight. Either elaborate why you should "Plan your implementation with SOLID principles in mind" or remove the paragraph